content/collections.md

```diff
@@ -0,0 +1,370 @@
+
+You can see from this example, that with relatively little code we've managed to restrict the format of a list significantly. You can read more in the [Simple Schema docs](http://atmospherejs.com/aldeed/simple-schema) about more complex things that can be done with schemas.
+
+### Checking documents against a schema
+
+Now we have a schema, how do we use it?
+
+The straightforward way to use a schema is by using the `check` package. We can write:
+
+```js
+const list = {
+  name: 'My list',
+  incompleteCount: 3
+};
+
+check(list, Lists.schema);
+```
+
+In this case, as the list is valid according to the schema, the `check()` line will run without problems. If however, we wrote:
+
+```js
+const list = {
+  name: 'My list',
+  incompleteCount: 3,
+  madeUpField: 'this should not be here'
+};
+
+check(list, Lists.schema);
+```
+
+// XXX: this isn't actually the case yet. Change this unless we update simple schema to do so..
+
+Then the `check()` call will throw a `Meteor.ValidationError` which contains details about what is wrong with the `list` document.
+
+### The `Meteor.ValidationError`
+
+What is a [`Meteor.ValidationError`](https://github.com/tmeasday/validation-error/)? It's a special error that is used in Meteor to indicate a user-input based error in modifying a collection. Typically, the details on a `ValidationError` are used to mark up a form with information about what inputs don't match the schema. In the "Methods and Forms" article, we'll see more about how this works.
+
+## Designing your data schama
+
+Now you are familiar with the basic API of Simple Schema, it's worth considering a few of the constraints of the Meteor system that can influence the design of your data schema. Although generally speaking you can build a Meteor data schema much like any Mongo data schema, there are some important differences.
+
+The most important consideration is due to the way that DDP communicates documents over the wire. The key thing to realize is that DDP sends changes to documents at the level of document *fields*. What this means is that if you have large and complex subfields on document that change often, DDP can send unnecessary changes over the wire.
+
+For instance, in "pure" Mongo you might design the schema so that each list document had a field called `todos` which was an array of todo items:
+
+```js
+Lists.schema = new SimpleSchema({
+  name: {type: String},
+  todos: {type: [Object]}
+});
+```
+
+The issue with this schema is that due to the DDP behaviour just mentioned, each change to *any* todo item in a list will require sending the *entire* set of todos for that list over the wire. This is because DDP has no concept of "change the `text` field of the 3rd item in the field called `todos`", simply "change the field called `todos` to (say) `[{text: 'first'}, {text: 'second'}]`".
+
+### Denormalization and multiple collections
```

Comment:
We might want to go over the concept of normalization vs denormalization here a little